content/partners/_index.md

```diff
@@ -1,8 +1,8 @@
 ---
 title : "Our partners"
 page_header_bg : "images/background/page-title-bg.jpg"
-date: 2024-11-06T20:41:24+02:00
+date: 2025-03-21T14:42:28+01:00
 description : "Meet our partners whose valuable support makes FOSS4G Europe 2025 in Mostar, Bosnia-Herzegovina a reality"
 draft : false
-type: "speaker"
+type: "partner"
```

Comment:
type should be "sponsor" (see comment from [PR18](https://github.com/foss4g-europe/2025-foss4g-europe.github.io/pull/18#discussion_r2005462601))